helper/testhelpers/corehelpers/corehelpers.go

```diff
@@ -210,13 +210,17 @@ type TestLogger struct {
 }
 
 func NewTestLogger(t testing.T) *TestLogger {
+	return NewTestLoggerWithSuffix(t, "")
+}
+
+func NewTestLoggerWithSuffix(t testing.T, logFileSuffix string) *TestLogger {
 	var logFile *os.File
 	var logPath string
 	output := os.Stderr
 
 	logDir := os.Getenv("VAULT_TEST_LOG_DIR")
 	if logDir != "" {
-		logPath = filepath.Join(logDir, t.Name()+".log")
+		logPath = filepath.Join(logDir, t.Name()+logFileSuffix+".log")
```

Comment:
nit: Would be nice to have a `_` between the name and suffix value if not ""?

Reply:
Good idea, I have added code that adds the underscore if necessary.